environments/aat.yml

```diff
@@ -89,6 +89,12 @@ cname:
   - name: "manage-org"
     ttl: 3600
     record: "hmcts-aat.azurefd.net"
+  - name: "afdverify.immigration-appeal"
```

Comment:
can you add to the bottom please, we have an issue with our code with adding to the middle (there's a PR open but we haven't rolled it out yet)

Reply:
I've added to the bottim.